src/main/scala/org/allenai/deep_qa/pipeline/QABackgroundToMC.scala

```diff
@@ -0,0 +1,69 @@
+package org.allenai.deep_qa.pipeline
+
+import com.mattg.pipeline.Step
+import com.mattg.util.FileUtil
+import com.mattg.util.JsonHelper
+import org.json4s._
+
+import org.allenai.deep_qa.data.DatasetReader
+
+import scala.collection.mutable.Map
+import scala.sys.process.Process
+import scala.sys.process.ProcessLogger
+
+/**
+  * This Step is a SentenceProducer that reads a file with MCReadingComprehensionInstances
+  * and removes the passages from them, turning it into a file with the format of a
+  * QuestionAnswerInstance. It does this by simply dropping the first column.
+  * Expected file format is "[passage][tab][question][tab][choices][tab][label]" or .
+  * "[index][tab][passage][tab][question][tab][choices][tab][label]".
+  * The output file format is "[question][tab][choices][tab][label]" or
+  * "[index][tab][question][tab][choices][tab][label]".
+ */
+class QABackgroundToMC(
+  val params: JValue,
+  val fileUtil: FileUtil
+) extends Step(None, fileUtil) with SentenceProducer {
+  implicit val formats = DefaultFormats
+  override val name = "No Passage MC"
+
+  val validParams = baseParams ++ Seq("sentences", "background", "output file")
+  JsonHelper.ensureNoExtras(params, name, validParams)
+
+  val sentenceProducer = SentenceProducer.create(params \ "sentences", fileUtil)
+  val sentencesFile = sentenceProducer.outputFile
+
+  val backgroundProducer = SentenceProducer.create(params \ "background", fileUtil)
+  val backgroundFile = backgroundProducer.outputFile
+
+  override val outputFile = JsonHelper.extractAsOption[String](params, "output file") match {
+    case None => sentencesFile.dropRight(4) + "_with_background_as_passages.tsv"
+    case Some(filename) => filename
+  }
+
+  override val inputs: Set[(String, Option[Step])] = Set((sentencesFile, Some(sentenceProducer)))
+  override val outputs = Set(outputFile)
+  override val paramFile = outputs.head.dropRight(4) + "_params.json"
+  override val inProgressFile = outputs.head.dropRight(4) + "_in_progress"
+
+  override def _runStep() {
+    fileUtil.mkdirsForFile(outputFile)
+    // make a map from indexes to paragraphs
+    val passageMap = fileUtil.mapLinesFromFile(backgroundFile, line => {
+      val fields = line.split("\t")
+      val index = fields(0)
+      val passage = fields.drop(1).mkString(" ")
+      (index, passage)
+    }).toMap
+
+    val outputLines = fileUtil.flatMapLinesFromFile(sentencesFile, line => {
+      val fields = line.split("\t")
+      val index = fields(0)
```

Comment:
`fields(0).toInt` here, to match the change above (assuming you actually make that change).